### DISCUSSION TITLE
css

### DISCUSSION THREAD

**Discussion by Parvanah** [Afghn-Girls-Tech-Team/Web-development#17] [Ideas]
i want to put all the provinces of Afghanistan in a list.what is your opinion in this case.

**Comment by KhatimaSadat** [comment]
your style was good just add the picture that you have added to the html design so that it show in other pc as well

**Comment by ShaqayeqRahmani** [comment]
it was not bad developer sahib

**Comment by Nooriaehsani** [comment]
you can add it
